lute/std/libs/fs.luau

```diff
@@ -10,13 +10,14 @@ local sys = require("@std/system")
 local fslib = {}
 
 export type handlemode = fs.HandleMode
-export type filehandle = fs.FileHandle
-export type filetype = fs.FileType
-export type filemetadata = fs.FileMetadata
+export type file = fs.FileHandle
+export type type = fs.FileType
+export type metadata = fs.FileMetadata
 export type directoryentry = fs.DirectoryEntry
 export type watchhandle = fs.WatchHandle
 export type watchevent = fs.WatchEvent
-export type pathlike = pathlib.pathlike
+
+type pathlike = pathlib.pathlike
```

Comment:
an aside, but we should _really_ name this type `path` if it's the one everyone is writing everywhere.